docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -74,10 +78,14 @@ spec:
       vlan: 22
 EOF
 ----
++
+* When `neutron-sriov-nic-agent` is running on the existing Compute nodes, the physical device mappings match the values that are defined in the `OpenStackDataPlaneNodeSet` custom resource (CR). For more information, see xref:pulling-configuration-from-tripleo-deployment_adopt-control-plane[Pulling the configuration from a {OpenStackPreviousInstaller} deployment].
 
-* When `neutron-sriov-nic-agent` is running on the existing Compute nodes, check the physical device mappings and ensure that they match the values that are defined in the `OpenStackDataPlaneNodeSet` custom resource (CR). For more information, see xref:pulling-configuration-from-tripleo-deployment_reviewing-configuration[Pulling the configuration from a {OpenStackPreviousInstaller} deployment].
-
-* Define the shell variables necessary to run the script that runs the fast-forward upgrade. Omit setting `CEPH_FSID`, if the local storage backend is going to be configured by Nova for Libvirt. The storage backend cannot be changed during adoption, and must match the one used on the source cloud:
+* The shell variables that are necessary to run the script that runs the fast-forward upgrade are defined:
++
+[NOTE]
+Do not set `CEPH_FSID` if the local storage back end is going to be configured by Nova for Libvirt. The storage back end must match the one used on the source cloud. You cannot change the storage back end during adoption.
```

Comment:
Maybe:
"Do not set a value for the `CEPH_FSID` parameter if the local storage back end is configured by Nova for `Libvirt`."

Should you say "Compute service" instead of just nova?

" The storage back end must match the source cloud storage back end. You cannot change the storage back end during adoption."

What do we mean by "match"? Are you using the same back end or it looks the same (same set up or vendor etc)?

Reply:
matches, like totally equals in the type and configuration